packages/runtime/src/useCases/transport/relationshipTemplates/CreateTokenQRCodeForOwnRelationshipTemplate.ts

```diff
@@ -52,7 +53,7 @@ export class CreateTokenQRCodeForOwnTemplateUseCase extends UseCase<CreateTokenQ
             return Result.fail(RuntimeErrors.relationshipTemplates.personalizationMustBeInherited());
         }
 
-        if (template.passwordProtection && request.passwordProtection !== template.passwordProtection) {
+        if (template.passwordProtection && !_.isEqual(request.passwordProtection, template.passwordProtection)) {
```

Comment:
I'd prefer an isEqual method on the PasswordProtection object. Comparing classes with lodash's isEqual should be avoided.

Reply:
I see, then some isEqual checks in the AttributeController need a JSON conversion first, I'm on it